packages/extension/src/companion/CompanionContent.tsx

```diff
@@ -59,10 +63,9 @@ export default function CompanionContent({
     }
 
     const { height } = el.getBoundingClientRect();
-    const topOffset = 7.5;
-    const remValue = height / 16;
-    const rem = `${remValue + topOffset}rem`;
-    setHeightRem(rem);
+    const topOffset = remToPx(COMPANION_TOP_OFFSET_REM);
```

Comment:
I suggest already storing the offset in pixels instead of calculating it at runtime

Reply:
Totally agree. I have now pushed the changes. 🚢